Header header
nist_core/rcs_stat stat

# poses associated with emove_move cmd
geometry_msgs/Pose current_pose
geometry_msgs/Pose target_pose

# file name and current line for emove_run cmd
string name
string line
string crcl

